src/components/MovieCard/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import { ThemeContext } from "styled-components";
+
+import PosterPlaceholder from "../shared/PhotoPlaceholder";
+import { Poster, Card, Date, Rating, StarIcon, Title, Content } from "./styles";
+import getIcon from "../../util/getIcon";
+
+export default function MovieCard({ movie, type, mediaType }) {
+    const cardDate = (date) => (date ? String(date).slice(0, 4) : "unknown");
```

Comment:
I think this should be outisde the function, since this needed in state change and is just formatting the date?